catalog/transformer/catalog-transformer-csv-queryresponse/src/test/java/ddf/catalog/transformer/csv/CsvMetacardTransformerTest.java

```diff
@@ -179,8 +182,8 @@ public void testBinaryAndObjectFieldsAreExcluded()
     List<String> attNames = Arrays.asList(lines.get(0).split(","));
     List<String> attValues = Arrays.asList(lines.get(1).split(","));
 
-    assertThat(attNames, contains(stringAtt));
-    assertThat(attValues, contains("stringVal"));
+    assertThat(attNames, containsInAnyOrder(stringAtt, MetacardType.METACARD_TYPE));
```

Comment:
✏️ By themselves, the orders of the values in these lines don't matter, but we probably want to assert that "stringVal" is in the same place as `stringAtt`, and the same for the metacard  type.

Reply:
Yeah, I took the shortcut solution knowing that it was only 2 items and not knowing which order they would come back in.  I'll go back and update it to be correct.